### PR TITLE
feat(strimzi-ci): fix ci with new version

### PR DESCRIPTION
**Pull request by ninjatux**

### CODE REVIEW

File: strimzi-kafka-connect/Dockerfile
Comment (dorsradek) [inline strimzi-kafka-connect/Dockerfile:1]: I think the FROM is incorrect(1st line)

Comment (ninjatux) [reply strimzi-kafka-connect/Dockerfile:1]: it is, I didn't pulled my own code before making this change 👍 👎 👍

**Review by dorsradek — APPROVED** [review]

File: strimzi-kafka-connect/Dockerfile
Comment (ninjatux) [reply strimzi-kafka-connect/Dockerfile:1]: no I'm stupid, I thought my previous PR was merged...

File: strimzi-kafka-connect/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
-FROM quay.io/strimzi/kafka:0.27.1-kafka-2.8.1
+FROM quay.io/strimzi/kafka:0.39.0-kafka-3.5.1
 LABEL maintainer="SRE Team <sre@pleo.io>"
 
 USER root:root
+
 RUN mkdir -p /opt/kafka/plugins/debezium
 RUN curl https://repo1.maven.org/maven2/io/debezium/debezium-connector-postgres/2.4.1.Final/debezium-connector-postgres-2.4.1.Final-plugin.tar.gz | tar xvz
```
Comment (dorsradek) [inline strimzi-kafka-connect/Dockerfile:7]: we could consider version 2.4.2 or 2.5.0

**Review by PassFaid — APPROVED** [review]

File: strimzi-kafka-connect/Dockerfile (hunk 1 above)
Comment (ninjatux) [reply strimzi-kafka-connect/Dockerfile:7]: where did you see that? I can see the latest marked as `final` is `2.4.1` here --> https://repo1.maven.org/maven2/io/debezium/debezium-connector-postgres/

Comment (dorsradek) [reply strimzi-kafka-connect/Dockerfile:7]: hmm 🤔 https://mvnrepository.com/artifact/io.debezium/debezium-connector-postgres

**Pull request merged**